src/main/scala/io/sdkman/changelogs/java/AdoptOpenJdkMigrations.scala

```diff
@@ -345,4 +347,178 @@ class AdoptOpenJdkMigrations {
+        MacOSX,
+        Some(AdoptOpenJDK)
+      ),
+      Version(
+        "java",
+        "11.0.7.j9-adpt",
+        "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_x64_windows_openj9_11.0.7_10_openj9-0.20.0.zip",
+        Windows,
+        Some(AdoptOpenJDK)
+      )
+    ).validate()
+      .insert()
+    Seq(Linux64, MacOSX, Windows).foreach(
+      removeVersion("java", "11.0.6.j9-adpt", _)
+    )
+  }
+
+  @ChangeSet(
+    order = "0013",
+    id = "0013-add_add_adoptopenjdk-hs_8_0_252",
+    author = "poad"
+  )
+  def migrate0013(implicit db: MongoDatabase) = {
+    List(
+      Version(
+        "java",
+        "8.0.252.hs-adpt",
+        "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u252-b09/OpenJDK8U-jdk_x64_linux_hotspot_8u252b09.tar.gz",
+        Linux64,
+        Some(AdoptOpenJDK)
+      ),
+      Version(
+        "java",
+        "8.0.252.hs-adpt",
+        "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u252-b09/OpenJDK8U-jdk_x64_mac_hotspot_8u252b09.tar.gz",
+        MacOSX,
+        Some(AdoptOpenJDK)
+      ),
+      Version(
+        "java",
+        "8.0.252.hs-adpt",
+        "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u252-b09/OpenJDK8U-jdk_x64_windows_hotspot_8u252b09.zip",
+        Windows,
+        Some(AdoptOpenJDK)
+      )
+    ).validate()
+      .insert()
+    Seq(Linux64, MacOSX, Windows).foreach(
+      removeVersion("java", "8.0.242.hs-adpt", _)
+    )
+  }
+
+  @ChangeSet(
+    order = "0014",
+    id = "0014-add_adoptopenjdk-j9_8_0_252",
+    author = "poad"
+  )
+  def migrate0014(implicit db: MongoDatabase) = {
+    List(
+      Version(
+        "java",
+        "8.0.252.j9-adpt",
+        "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u252-b09_openj9-0.20.0/OpenJDK8U-jdk_x64_linux_openj9_8u252b09_openj9-0.20.0.tar.gz",
+        Linux64,
+        Some(AdoptOpenJDK)
+      ),
+      Version(
+        "java",
+        "8.0.252.j9-adpt",
+        "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u252-b09_openj9-0.20.0/OpenJDK8U-jdk_x64_mac_openj9_8u252b09_openj9-0.20.0.tar.gz",
+        MacOSX,
+        Some(AdoptOpenJDK)
+      ),
+      Version(
+        "java",
+        "8.0.252.j9-adpt",
+        "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u252-b09_openj9-0.20.0/OpenJDK8U-jdk_x86-32_windows_openj9_8u252b09_openj9-0.20.0.zip",
+        Windows,
+        Some(AdoptOpenJDK)
+      )
+    ).validate()
+      .insert()
+    Seq(Linux64, MacOSX, Windows).foreach(
+      removeVersion("java", "8.0.242.j9-adpt", _)
+    )
+  }
+
+  @ChangeSet(
+    order = "0015",
+    id = "0015-add_adoptopenjdk-j9_14_0_1",
+    author = "poad"
+  )
+  def migrate0015(implicit db: MongoDatabase) = {
```

Comment:
Is there a `hs` variant available for `14.0.1`?

Reply:
@marc0der 
No hotspot has been released for 14.0.1 yet.

Oops!
According to AdoptOpenJDK's Slack, it looks like it's going to be released today.

Reply:
@marc0der 
14.0.1 of hotspot has been released, so I added it.
commit is https://github.com/sdkman/sdkman-db-migrations/pull/249/commits/1c08c64a63195c532650fba447822bafedda1453.